test/bigtest/tests/units/AllJobLogs-test.js

```diff
@@ -0,0 +1,187 @@
+import React from 'react';
+import { BrowserRouter as Router } from 'react-router-dom';
+import { noop } from 'lodash';
+import { expect } from 'chai';
+import Pretender from 'pretender';
+import { cleanup } from '@bigtest/react';
+import {
+  describe,
+  before,
+  beforeEach,
+  afterEach,
+  it,
+} from '@bigtest/mocha';
+
+import {
+  mountWithContext,
+  buildResources,
+  buildMutator,
+} from '@folio/stripes-data-transfer-components/interactors';
+import CalloutInteractor from '@folio/stripes-components/lib/Callout/tests/interactor';
+import { StripesContext } from '@folio/stripes-core/src/StripesContext';
+import { Paneset } from '@folio/stripes/components';
+import commonTranslations from '@folio/stripes-data-transfer-components/translations/stripes-data-transfer-components/en';
+
+import translations from '../../../../translations/ui-data-export/en';
+import { allLogsPaneInteractor } from '../../interactors';
+import { AllJobLogsViewComponent } from '../../../../src/components/AllJobLogsView';
+import {
+  OverlayContainer,
+  translationsProperties,
+} from '../../helpers';
+import { logJobExecutions } from '../../network/scenarios/fetch-job-executions-success';
+
+const {
+  searchResults,
+  header,
+} = allLogsPaneInteractor.searchAndSort;
+const { getCellContent } = searchResults;
+const getUser = row => logJobExecutions[row].runBy;
+
+describe('All job logs view', () => {
```

Comment:
Please, use a component name to describe these tests (this approach is used in other places)